src/NotificationMessage.ts

```diff
@@ -0,0 +1,23 @@
+export class NotificationMessage {
```

Comment:
** PERSONAL OPINION KLAXON **
I don't think there is much value in this being a class (I almost never do) 

```
interface NotificationMessage {
    message: string;
    url: string;
    details: string;
}
```
will do just fine here because the intention is to describe the shape of the data not to provide behaviour. 

Why do I think you should just about _always_ favour interfaces over classes? Because I think that mixing data and behaviour (as a class encourages you to do) is just about always a bad idea. Functions for the win.

Reply:
see https://github.com/efcloud/el-msteams-post-action/pull/1#discussion_r386245869 --> should be done